docs/markdown/Reference-manual.md

```diff
@@ -2463,7 +2463,10 @@ the following methods:
   positional argument compiles and links, you can specify external
   dependencies to use with `dependencies` keyword argument, `code` can
   be either a string containing source code or a `file` object
-  pointing to the source code.
+  pointing to the source code.  If the `file` object's suffix does
+  not match the compiler object's language, the compiler corresponding
+  to the suffix is used to compile the source, while the target of
+  the `links` method is used to link the resulting object file.
```

Comment:
This should mention that this is only with 0.59 onwards. And should this say "while the compiler of the `links` method" rather than "while the target of the `links` method"?

Reply:
0.60.0, right? 😃

Reply:
I used "target" as in "what's to the left of the dot". Added the "since" now.